src/utils.py

```diff
@@ -69,8 +71,17 @@ def get_webdriver() -> WebDriver:
 
     # downloads and patches the chromedriver
     # if we don't set driver_executable_path it downloads, patches, and deletes the driver each time
-    driver = uc.Chrome(options=options, driver_executable_path=driver_exe_path, version_main=version_main,
-                       windows_headless=windows_headless)
+    # ToDo: seleniumwire supports proxy through seleniumwire_options
```

Comment:
This potentially closes: https://github.com/FlareSolverr/FlareSolverr/issues/657